sdk/core/core-tracing/src/createSpan.ts

```diff
@@ -48,6 +49,31 @@ export function isTracingDisabled(): boolean {
   return Boolean(azureTracingDisabledValue);
 }
 
+function pluckOptions<
+  T extends { tracingOptions?: OperationTracingOptions } & { spanOptions?: SpanOptions }
```

Comment:
this is super confusing, but we don't have access to `OperationOptions` and I wanted to be able to continue passing customization to newly created spans. So now you can do something like:

```
const span = createSpan("deleteConfigurationSetting", {...operationOptions, spanOptions: { spanKind: SpanKind.SERVER } });
```